helix-view/src/input.rs

```diff
@@ -375,6 +363,16 @@ mod test {
         assert!(str::parse::<KeyEvent>("FU").is_err());
         assert!(str::parse::<KeyEvent>("123").is_err());
         assert!(str::parse::<KeyEvent>("S--").is_err());
+        assert!(str::parse::<KeyEvent>("S-percent").is_err());
+    }
+
+    #[test]
+    fn parsing_unsupported_named_keys() {
+        assert!(str::parse::<KeyEvent>("lt").is_err());
+        assert!(str::parse::<KeyEvent>("gt").is_err());
+        assert!(str::parse::<KeyEvent>("plus").is_err());
+        assert!(str::parse::<KeyEvent>("percent").is_err());
+        assert!(str::parse::<KeyEvent>("semicolon").is_err());
```

Comment:
Rather than just testing these keys are now unsupported, shouldn't we also test that they get parsed correctly by the new syntax?

Reply:
@archseer Great catch! I intended to do this, but only added a test for `percent`. I created a new PR to add in the additional tests. #2694